AI/Nullkiller/CMakeLists.txt

```diff
@@ -120,24 +117,24 @@ set(Nullkiller_HEADERS
 		AIGateway.h
 )
 
+if(NOT ENABLE_STATIC_AI_LIBS)
+	list(APPEND Nullkiller_SRCS main.cpp StdInc.cpp)
+endif()
```

Comment:
Maybe ifdef in code would be better?

Reply:
don't see any advantages, could you explain?

Reply:
Not really, other than simplifying CMakeLists a bit.
Feel free to ignore.